mutmut/__main__.py

```diff
@@ -587,20 +602,34 @@ def run_mutation_tests(config, mutations_by_file):
         run_mutation_tests_for_file(config, file_to_mutate, mutations)
 
 
-def read_coverage_data(use_coverage):
+def read_coverage_data():
     """
     :type use_coverage: bool
     :rtype: CoverageData or None
     """
-    if use_coverage:
-        print('Using coverage data from .coverage file')
-        # noinspection PyPackageRequirements,PyUnresolvedReferences
-        from coverage import Coverage
-        cov = Coverage('.coverage')
-        cov.load()
-        return cov.get_data()
-    else:
-        return None
+    print('Using coverage data from .coverage file')
+    # noinspection PyPackageRequirements,PyUnresolvedReferences
+    from coverage import Coverage
+    cov = Coverage('.coverage')
+    cov.load()
+    return cov.get_data()
+
+
+def read_patch_data(use_patch_file):
```

Comment:
We should rename the arg `use_patch_file` to `patch_file_path` so that it is more self descriptive.